### PR TITLE
AMP-52285 Add pod dependency to Unity iOS SDK

### PR DESCRIPTION
**Pull request by dnshi**
This PR is to use Unity iOS Pod file to import dependency. 

# Screenshot
image
image

### CODE REVIEW

File: Assets/Editor/AmplitudeDependencies.xml [hunk 1]
```diff
@@ -18,6 +18,6 @@
   </androidPackages>
 
   <iosPods>
-  
+    <iosPod name="AnalyticsConnector" version="~> 1.0.0" />
```
[the remark this answers is not in the document]
Comment (dnshi) [reply Assets/Editor/AmplitudeDependencies.xml:21]: Customer would need [External Dependency Manager for Unity](https://github.com/googlesamples/unity-jar-resolver), which is required in order to build iOS project.

[the remark this answers is not in the document]
Comment (qingzhuozhen) [reply Assets/Editor/AmplitudeDependencies.xml:21]: @dnshi So if they install our [unity package](https://amplitude.atlassian.net/wiki/spaces/GOV/pages/1102544936/Unity#Installing-Unity-SDK-onto-a-New-Game-Project), would they be auto imported? Or would there be some extra steps required?

[the remark this answers is not in the document]
Comment (dnshi) [reply Assets/Editor/AmplitudeDependencies.xml:21]: I am creating a new game project to test it out.

[the remark this answers is not in the document]
Comment (dnshi) [reply Assets/Editor/AmplitudeDependencies.xml:21]: @qingzhuozhen I started from a new project and imported our Unity SDK. The external dependency manager is required in order to create a Podfile.

What external dependency manager does is that, it integrates with Cocoapods and use `pod` to generate a Podfile. 

I don't think all the customers install the dependency manager, so we need to update our developer document to explain it.

**Comment by dnshi** [comment]
@qingzhuozhen Added "External Package Manager". I will create a separate PR to update version number and change log.

**Pull request merged**